### ISSUE TITLE
Freezing

### ISSUE THREAD

**Issue by LuisBurdallo** [lowfuel/progrockdiffusion#148]
Any clue if anyone else get it to freeze sometimes when starting a new job. It get's stuck at 0% and no meters work. I just closed the command window and it run into a BSOD saying video scheduler internal error. It restarted the computer automatically.

**Comment by lowfuel** [comment]
I haven't seen that personally, no. A few questions:
- Does it happen every time?
- Or every time with certain settings? (please share them if so)
- Can you run nvidia-smi and paste the results here?

**Comment by LuisBurdallo** [comment]
This is weird like never before with some other jobs it started after a few minutes. I didn't close the windows this time. I run Nvidia-smi but I don't know where to find the output.

**Comment by LuisBurdallo** [comment]
This is the only thing different from some other prompts I was using earlier.

trending on artstation:2', 'anime:3', 'car:-1', 'people:-1', 'dof:-1', 'blur:-1']

**Comment by lowfuel** [comment]
Would you mind providing the full settings file? I just tried with all those prompt weights and it worked fine here, so it might be some combination of that prompt with other settings.
Alternatively, you could use PRD's default settings.json file, but with your full prompt, and see if that has the same issue. 

Let me know!

**Comment by LuisBurdallo** [comment]
I just pasted the same prompt into the original settings file and it does the same thing. So after a few tests:

This freezes:
["A beautiful painting of a Castle in the Scottish Highlands, underexposed and overcast, by Asher Brown Durand, trending on artstation:5", "anime", "car:-1", "people:-1", "dof:-1", "blur:-1",
]
This does not:
["A beautiful painting of a Castle in the Scottish Highlands, underexposed and overcast, by Asher Brown Durand, trending on artstation:5", "anime:3, car:-1, people:-1, dof:-1, blur:-1"
 ]

Not entirely sure if it means the same thing both prompts. Any clue why it could be?

**Comment by lowfuel** [comment]
It's working here for me. Are you running the latest progrockdiffusion files?

**Comment by lowfuel** [comment]
By the way, that second version of your prompt will get cut off after anime:3 as the parser basically stops after a weight value and considers that the end of the prompt. A given prompt (a part with quotes) can only have one weight.

Which might be an interesting clue. If you are still having problems on the latest files, you might try the freezing version of the prompt but remove the extra prompts one at a time until it works. I wish I could reproduce it or I'd do that here. :/

**Comment by LuisBurdallo** [comment]
Yes I updated yesterday. I followed this installation method. https://docs.google.com/document/d/1FlH3D4dL6OGN2_hlfzT5_AbAQbH-woDARFCb1LLmbr8/edit
I had to reinstall attr after numpy because it was asking me for the library. Let me know if everything looks good in that installation guide.

I am currently trying to reproduce my own bug. The thing is it should work or not if it's based on syntax issues. Why would it get stuck for several minutes and suddenly continue? That's what got me puzzled.

**Comment by lowfuel** [comment]
I'm not sure, it is strange. As for that install guide I guess it looks okay, but I haven't tried it personally so I can't vouch for it. Might be worth comparing it to the instructions in the README to make sure nothing got left out or is coming from the wrong place.

Can you paste the full output when you run PRD, just so I can see all the initial setup and what it's saying? Remember to put three ` characters before it so Github formats it properly.

**Comment by LuisBurdallo** [comment]
Right now everything seems to work. Here is the output when running one of the test I was doing now.
```
PROG ROCK DIFFUSION
-------------------

Parsing settings.json

Eta set automatically to: 0.75
Clamp_max automatically set to [0.05]*1000
clip_guidance_scale set automatically to: [7277]*1000
Converted cutn_batches to schedule.

Prompt(s) with randomizers:
  0: ['A beautiful ultradetailed anime illustration of a city street by beeple, makoto shinkai, and thomas kinkade, anime art wallpaper 4k, trending on artstation:3', 'anime', 'car:-1', 'dof:-1', 'blur:-1']


Pytorch is using device: cuda:0
Loading diffusion model details from diffusion_models.json
Using Diffusion Model: 512x512_diffusion_uncond_finetune_008100

Loading CLIP Models:

--ViTB32
--ViTB16
--RN50

Starting Run: Default(9) at frame 0
Using seed 3837811193

Starting batch!
Skipping 0 steps

Prompt for step 0: ['A beautiful ultradetailed anime illustration of a city street by beeple, makoto shinkai, and thomas kinkade, anime art wallpaper 4k, trending on artstation:3', 'anime', 'car:-1', 'dof:-1', 'blur:-1']
Image 1 of 1: :  64%|█████████████████████████████████████████▏                      | 161/250 [06:28<03:34,  2.41s/it]

**Comment by lowfuel** [comment]
I wonder if it was downloading something. If it happens again please check your network activity and let me know. Most of the downloads that PRD does display a message, but some of the underlying tools also download models and such, and they may or may not display something to that effect.
